myproject/blog/views.py

```diff
@@ -37,4 +37,9 @@ def get_context_data(self, **kwargs: Any):
         context = super().get_context_data(**kwargs)
         post = self.get_object()
         context['comments'] = post.comments.all().order_by("created_at")
-        return context
\ No newline at end of file
+        return context
+    
+class BloggerList(generic.ListView):
+     model = Author
+     template_name = "blog/blogger_list.html"
+     context_object_name = "bloggers"
```

Comment:
Make sure you have pagination in all the list view. Please add pagination in all the list view.